src/main/scala/rps/Game.scala

```diff
@@ -0,0 +1,51 @@
+package rps;
+
+object Game {
+  def play(): Unit = {
+
+    /* Ask for user input */
+    val userMove = readLine(
+      "What's your move?\n" +
+        "0: " + returnMoveIcon("0") + "\n" +
+        "1: " + returnMoveIcon("1") + "\n" +
+        "2: " + returnMoveIcon("2") + "\n\n" +
+        "> ")
```

Comment:
This is fine, but try to rewrite this using:

- [string interpolation](https://docs.scala-lang.org/overviews/core/string-interpolation.html) (`s""`)
- multiline strings with `stripMargin` (https://alvinalexander.com/scala/how-to-create-multiline-strings-heredoc-in-scala-cookbook)